.env

```diff
@@ -0,0 +1,7 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
Always put `.env` or `.env.local` files on the `.gitignore`. Never push your environmental variables on the git.